src/main/lib/string-utils.js

```diff
@@ -0,0 +1,7 @@
+const split = (input, delimeter = ',') => {
+  return input == null ? [] : input
```

Comment:
Bit unexpected to me. I'd rather expect to get null if input is null.

Reply:
Hmm, it leads to a bit cleaner code that way, as I don't need to handle the null scenario in the list file as my contract required array. 

But happy to change if you believe it's confusing.

Reply:
Ok removed the support for null / undefined. This is just going to fail when null undefined is passed, less test for this private API.